knt-backend/app/models.py

```diff
@@ -2,42 +2,98 @@
 from flask_restful import Resource
 from flask import request
 
+""" USERS """
+
+
 class User(db.Model):
     id = db.Column(db.Integer, primary_key=True)
     first_name = db.Column(db.Text)
     last_name = db.Column(db.Text)
     nickname = db.Column(db.Text)
```

Comment:
Is this to be the reference, like the VUnet ID? I would rather name it like so, it should be a bit more clear.

Reply:
the ID is just the primary key generated by the database. I will include a vunet_id field as well (forgot about it).